src/main/java/se/sundsvall/casedata/api/model/validation/enums/MessageType.java

```diff
@@ -1,5 +1,5 @@
 package se.sundsvall.casedata.api.model.validation.enums;
 
 public enum MessageType {
-	SMS, EMAIL, WEBMESSAGE
+	SMS, EMAIL, WEBMESSAGE, MYPAGES
```

Comment:
When the same logic was implemented in SM it was concluded that `MYPAGES` was not needed and using `WEBMESSAGE`for the new MyPages stuff was sufficient. @AndreSoderlund might have some more insight as he was the one implementing.

Reply:
Removed MYPAGES